doc/modules/ensemble.rst

```diff
@@ -150,6 +150,19 @@ In contrast to the original publication [B2001]_, the scikit-learn
 implementation combines classifiers by averaging their probabilistic
 prediction, instead of letting each classifier vote for a single class.
 
+A competitive alternative to random forests are
+:ref:`histogram_based_gradient_boosting` (HGBT) models:
+
+- Building trees: Random forests typically rely on deep trees (that overfit individually) which uses much computational resources, as they require several splittings and evaluations of candidate splits. Boosting models build shallow trees (that underfit individually) which are faster to fit and predict.
+
+- Sequential boosting: In histogram-based gradient boosting (HGBT), the decision trees are built sequentially, where each tree is trained to correct the errors made by the previous trees. This is achieved by assigning higher weights to the training instances that were misclassified in the previous iteration, which helps the next tree to focus on those instances and improve their classification. This process is repeated iteratively until a predefined number of trees are built or until the model's performance stops improving. 
+
+  In contrast, random forests (RF) use an ensemble of decision trees, where each tree is built independently and is not influenced by the other trees. The final prediction is made by taking the majority vote of all the trees in the ensemble. While RF can achieve high accuracy, they may require a larger number of trees to achieve the same level of accuracy as HGBT. This is because RF may not be able to correct errors made by previous trees as effectively as HGBT, which can lead to a larger number of trees needed to achieve the desired level of accuracy.
+
+- Efficient binning: Efficient binning: HGBT uses an efficient binning algorithm that can handle large datasets with a high number of features. The binning algorithm can pre-process the data to speed up the subsequent tree construction (see `Why-it's-faster <https://scikit-learn.org/stable/modules/ensemble.html#why-it-s-faster>`__). In contrast, the scikit-learn implementation of random forests does not use binning and relies on exact splitting, which can be computationally expensive.
```

Comment:
For the reference, sphinx supports reStructured text:

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
Sure, made the changes.